### PR TITLE
Show pegin wallets based on feature flags

### PR DESCRIPTION
**Pull request by lserra-iov**

### CODE REVIEW

**Review by ronaldsg20 — APPROVED** [review]

File: src/common/components/exchange/SelectBitcoinWallet.vue [hunk 1]
```diff
@@ -60,8 +61,13 @@ export default {
     const router = useRouter();
     const storeConstants = constants;
     const environmentContext = EnvironmentContextProviderService.getEnvironmentContext();
+    const getFeature = useGetter<(name: FeatureNames) => Feature>('web3Session', constants.SESSION_GET_FEATURE);
 
-    const wallets = ref(walletConf.wallets);
+    const wallets = computed(() => walletConf.wallets.filter((wallet) => {
+      const walletName = wallet.constant as BtcWallet;
```
Comment (alexjavabraz) [inline src/common/components/exchange/SelectBitcoinWallet.vue:67]:
```suggestion
      const walletConfJsonConstant = wallet.constant as BtcWallet;
```

File: src/common/components/exchange/SelectBitcoinWallet.vue [hunk 2]
```diff
@@ -60,8 +61,13 @@ export default {
...
+      const flag = getFeature.value(FeatureNames[walletName]);
```
Comment (alexjavabraz) [inline src/common/components/exchange/SelectBitcoinWallet.vue:68]:
```suggestion
      const flag = getFeature.value(FeatureNames[walletConfJsonConstant]);
```

**Review by alexjavabraz — CHANGES_REQUESTED** [review]
Just to clarify the source of that field, in order to make sure the env_variable name will be the correct one.

**Review by alexjavabraz — APPROVED** [review]
LGTM

**Review by annipi — APPROVED** [review]
LGTM!

**Review by leoiovlabs — APPROVED** [review]
LGTM

**Pull request merged**